File: linda-core/src/main/java/it/unibo/coordination/linda/core/InspectablePredicativeTupleSpace.java
package it.unibo.coordination.linda.core;

public interface InspectablePredicativeTupleSpace<T extends Tuple, TT extends Template, K, V>
        extends PredicativeTupleSpace<T, TT, K, V>, InspectableTupleSpace<T, TT, K, V> {

}

File: linda-core/src/main/kotlin/it/unibo/coordination/linda/core/InspectablePredicativeTupleSpace.kt
package it.unibo.coordination.linda.core

interface InspectablePredicativeTupleSpace<T : Tuple<T>, TT : Template<T>, K, V, M : Match<T, TT, K, V>>
    : PredicativeTupleSpace<T, TT, K, V, M>,
        InspectableLindaTupleSpace<T, TT, K, V, M>
